.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -454,8 +454,8 @@ jobs:
           node-count: 6
           reserved-seats: 3
           non-reserved-seats: 3
-          follow-up-finalization-check: true
-        timeout-minutes: 20
+          follow-up-finalization-check: false
```

Comment:
With this check turned off this PR makes no sense. :/

Reply:
Agreed, I am just checking if turning it off will make this pass (99% it will). Further investigation ongoing.